R/rounding.qmd

```diff
@@ -2,9 +2,15 @@
 title: "Rounding in R"
 ---
 
-The **round()** function in Base R will round to the nearest whole number and 'rounding to the even number' when equidistant, meaning that exactly 12.5 rounds to the integer 12.
+## round from R base
+The **round()** function in Base R will round to the nearest whole number and 'rounding to the even number' when equidistant, meaning that exactly 12.5 rounds to the integer 12. However, this is dependent on OS services and on representation error (since e.g. 0.15 is not represented exactly, the rounding rule applies to the represented number and not to the printed number, and so round(0.15, 1) could be either 0.1 or 0.2).
+```{R, eval=FALSE}
```

Comment:
To action Christina's comment you can try the suggestion below, eval=True is the default

```suggestion
```{r}
```